python/dolfinx/wrappers/mesh.cpp

```diff
@@ -268,8 +268,8 @@ void mesh(py::module& m)
       .def_property_readonly("cell_type", &dolfinx::mesh::Topology::cell_type)
       .def("cell_name", [](const dolfinx::mesh::Topology& self)
            { return dolfinx::mesh::to_string(self.cell_type()); })
-      .def("mpi_comm", [](dolfinx::mesh::Mesh& self)
-           { return MPICommWrapper(self.mpi_comm()); });
+      .def_property_readonly("mpi_comm", [](dolfinx::mesh::Mesh& self)
```

Comment:
Is this near zero-cost? It's been a function for a long time.

Reply:
It should just be some messing about with pointers.

Everything was a function in the first version of the pybind11 wrappers. We should, at some point, check the whole interface for what should be a property.